src/java.base/share/classes/jdk/internal/vm/ThreadDumper.java

```diff
@@ -305,9 +310,12 @@ private static void dumpThreads(ThreadContainer container, JsonWriter jsonWriter
      * Write a thread to the given JSON writer.
      * @throws UncheckedIOException if an I/O error occurs
      */
-    private static void dumpThread(Thread thread, JsonWriter jsonWriter) {
+    private static boolean dumpThread(Thread thread, JsonWriter jsonWriter) {
         Instant now = Instant.now();
         ThreadSnapshot snapshot = ThreadSnapshot.of(thread);
+        if (snapshot == null) {
+            return false; // Terminated
```

Comment:
This is okay. Do you mind change these two to say "thread terminated"?  We will eventually replace this as it's only a temporary that this scenario is even possible.

Reply:
Sure!